atlasexec/atlas.go

```diff
@@ -384,7 +400,7 @@ func (c *Client) Version(ctx context.Context) (*Version, error) {
 }
 
 // runCommand runs the given command and returns its output.
-func (c *Client) runCommand(ctx context.Context, args []string, vs ...validator) (io.Reader, error) {
+func (c *Client) runCommand(ctx context.Context, args []string, vs ...validator) (io.Reader, int, error) {
```

Comment:
For example:
if stderr > 0, returns its content. Otherwise, return if exit-code <> 1, return `errExitCode = errors.New("atlasexec: atlas command exited code 1")`. Then, catch that in the caller and handle accordingly.